include/freertos/FreeRTOSConfig.h

```diff
@@ -137,17 +137,38 @@ standard names - or at least those used in the unmodified vector table. */
  *************************************************************************** */
 #elif defined (GCC_ARMCM0)
 
+#define configCPU_CLOCK_HZ             ( cpu_clock_hz )
+#define configMINIMAL_STACK_SIZE       ( ( unsigned short ) 256 )
+#define configTOTAL_HEAP_SIZE          ( ( size_t ) ( 7000 ) )
+#define configTIMER_TASK_STACK_DEPTH   256
+
+
+#define configKERNEL_INTERRUPT_PRIORITY         255
+/* !!!! configMAX_SYSCALL_INTERRUPT_PRIORITY must not be set to zero !!!!
+See http://www.FreeRTOS.org/RTOS-Cortex-M3-M4.html. */
+#define configMAX_SYSCALL_INTERRUPT_PRIORITY     0xa0 /* equivalent to 191, or priority 5. */
```

Comment:
is there such a thing for Cortex-M0?
I thought interrupt priorities are only there on M3/M4.

Reply:
The Cortex-M0 does have interrupt priorities, however, interrupts can only be globally disabled, not by priority. I removed these two defines, they are unnecessary.